From 6074e95a07ab23cfe47fb33ba7c0effe4178c68b Mon Sep 17 00:00:00 2001
From: Tim Snyder <timothy.snyder@sifive.com>
Date: Wed, 29 Jun 2022 14:49:25 +0000
Subject: [PATCH 2/3] change -std=c11 to -std=gnu11 for aligned_alloc

in conda gcc 10.3 toolchain -std=c11 isn't enough to make aligned_alloc()
available. Switching to -std=gnu11 fixes.
---
 share/wake/lib/gcc_wake/gcc.wake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/share/wake/lib/gcc_wake/gcc.wake b/share/wake/lib/gcc_wake/gcc.wake
index c08b223d..15d1ba32 100644
--- a/share/wake/lib/gcc_wake/gcc.wake
+++ b/share/wake/lib/gcc_wake/gcc.wake
@@ -17,7 +17,7 @@ package gcc_wake
 
 export def cpp11Flags    = ("-std=c++11", Nil)
 export def cpp14Flags    = ("-std=c++14", Nil)
-export def c11Flags      = ("-std=c11", Nil)
+export def c11Flags      = ("-std=gnu11", Nil)
 export def debugCFlags   = ("-Wall", "-Wextra", "-O0", "-g", "-pg", Nil)
 export def debugLFlags   = ("-g", "-pg", Nil)
 export def releaseCFlags = ("-Wall", "-O2", Nil)
-- 
2.35.3

